scripts/init_app_dir.sh

```diff
@@ -29,6 +32,24 @@ if [[ -z "${IN_NIX_SHELL}" ]]; then
 	cp -P /usr/lib/x86_64-linux-gnu/libgst* "${APP_DIR}/usr/lib/"
 	cp -r /usr/lib/x86_64-linux-gnu/gstreamer-1.0 "${APP_DIR}/usr/lib/"
 	cp -r /usr/lib/x86_64-linux-gnu/gstreamer1.0 "${APP_DIR}/usr/lib/"
+
+	echo "Bundling pcsc-lite 2.2.3..."
+	cp -L /usr/local/lib/x86_64-linux-gnu/libpcsclite.so* "${APP_DIR}/usr/lib/"
+	cp -L /usr/local/lib/x86_64-linux-gnu/libpcsclite_real.so* "${APP_DIR}/usr/lib/"
+	cp -L /usr/local/lib/x86_64-linux-gnu/pkgconfig/libpcsclite.pc "${APP_DIR}/usr/lib/"
+
+	chmod 755 "${APP_DIR}/usr/lib/libpcsclite.so"*
+	chmod 755 "${APP_DIR}/usr/lib/libpcsclite_real.so"*
+	chmod 755 "${APP_DIR}/usr/lib/libpcsclite.pc"
+
+	echo "Bundling pcscd..."
+	cp -L "/usr/local/sbin/pcscd"* "${APP_DIR}/usr/sbin/"
+	chmod 755 "${APP_DIR}/usr/sbin/pcscd"*
+
+	if [ -f "/usr/lib/x86_64-linux-gnu/libusb-1.0.so.0" ]; then
+		cp -L /usr/lib/x86_64-linux-gnu/libusb-1.0.so.0* "${APP_DIR}/usr/lib/" || true
+	fi
```

Comment:
`libusb` in linuxdeployqt exclude list for some reasone:
https://github.com/probonopd/linuxdeployqt/blob/master/tools/linuxdeployqt/excludelist.h

probably using system libs is better, than bundling

Reply:
also here
https://github.com/AppImageCommunity/pkg2appimage/blob/1eceb306077e51b991e987690b364e11971b44b3/excludelist#L189

Reply:
hmm let me remove `libusb` and check.

Reply:
yeah system `libusb` also works well. Thanks for the suggestion.